src/js/main.js

```diff
@@ -181,12 +181,12 @@ $(document).ready(function() {
       };
       if (!finsecImagesCompleted) {
         if (scrollActivationPoint > finsecImagesOffset) {
-          $('#finsec .images img:nth-last-child(3)').addClass('active');
-          setTimeout(function() {
-            $('#finsec .images img:nth-last-child(2)').addClass('active');
+          $('#finsec .images #iphone').addClass('active');
```

Comment:
Naming it `#iphone` instead of `nth-last-child(3)` is a bit clearer for the next dev working on this. As well less brittle. Imagine I add an element, then the `nth` selectors would break.

Reply:
makes sense :)